core/modules/nfv_ctrl.cc

```diff
@@ -123,6 +132,9 @@ int NFVCtrl::NotifyRCoreToWork(cpu_core_t core_id, int q_id) {
     bess::ctrl::sw_q_state[q_id]->down_core_id = i;
     return 0;
   }
+  // No RCores found. System Overloaded! Drop packets
+  sw_q_to_drop_.push_back(bess::ctrl::sw_q[q_id]);
```

Comment:
Adding a queue to std::vector can cause race condiions. I can do a quick fix on it.